closed, errors, encoding added to stdout and stderr

rpuntaie opened the pull request.

rpuntaie committed:
closed, errors, encoding added to stdout and stderr

brammool commented:
Roland Puntaier wrote:

> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/vim/vim/pull/622
> 
> -- Commit Summary --
> - closed, errors, encoding added to stdout and stderr

Can you please explain what this changes?

## 

hundred-and-one symptoms of being an internet addict:
116. You are living with your boyfriend who networks your respective
     computers so you can sit in separate rooms and email each other

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\
\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\            help me help AIDS victims -- http://ICCF-Holland.org    ///

rpuntaie committed:
test86.ok and test87.ok

rpuntaie commented:
Some time back I added some fields to vim's stdout and stderr. Later on I executed some code that reported AttributeError "errors". Then I got request by Keith Dart, that "closed()" was missing. In addition I've added "encoding".
